### ISSUE TITLE
"Space close" typo

### ISSUE THREAD

**Issue by zgronska** [mblode/vscode-twig-language-2#105]
Despite the "space close" option being enabled, the space is not being inserted at the end of self-closing HTML tags. I noticed there is a typo in the "space close" option assignment: 
https://github.com/mblode/vscode-twig-language-2/blob/f62292b533e7c3628c3039214edf45869f8d16c9/src/extension.js#L69

Could that be the reason? 😊

**Comment by mblode** [comment]
Good catch — that exact typo (`config.spaceSlose`) was fixed on `master` in a later commit; `space_close` is now wired to the `spaceClose` setting correctly. Closing as resolved. (Note the formatter itself is being reworked off prettydiff — see #117.)

**Comment by mrleblanc101** [comment]
@mblode Could this get published before the complete rewrite off pretty-diff ?

**Comment by mblode** [comment]
The settings descriptions are corrected in 0.12.0. Thanks for flagging this.
